app/src/components/ReviewCard/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import Stack from "@mui/material/Stack";
+import Rating from "@mui/material/Rating";
+import IconButton from "@mui/material/IconButton";
+import ThumbUpIcon from "@mui/icons-material/ThumbUp";
+import useAuth from "../../hooks/useAuth";
+
+const ReviewCard = ({ review }) => {
+  const [liked, setLiked] = useState(false);
+  const { user } = useAuth();
+  return (
+    <Stack
+      key={review.review_id}
+      p={2}
+      border="2px solid #Dde6ea"
+      borderRadius={2}
+      spacing={1}
+      position="relative"
+    >
+      <Stack className="reviews" direction="row" spacing={2} mb={1}>
+        <Box display="flex" alignContent="center">
+          <Typography mr={1}>Cleanliness</Typography>
+          <Rating size="small" readOnly value={review.cleanliness_rating} />
+        </Box>
+        <Box display="flex" alignContent="center">
+          <Typography mr={1}>Admin Helpfulness</Typography>
+          <Rating
+            size="small"
+            readOnly
+            value={review.admin_helpfulness_rating}
+          />
+        </Box>
+      </Stack>
+      <Typography>"{review.comment}"</Typography>
+      <Typography>— Happy Hippo</Typography>
+      <Box
+        display="flex"
+        alignItems="center"
+        sx={{ position: "absolute", bottom: 4, right: 10 }}
+      >
+        <IconButton
+          color={liked ? "primary" : undefined}
+          disabled={!user}
+          sx={{ mr: "2px" }}
+          onClick={() => setLiked((prevLiked) => !prevLiked)}
```

Comment:
I like the way you have set up the thumbs up button. The only thing that's missing is that the number should update beside the thumb, and we should also persist the state of the helpfulness to the backend (most likely through an update endpoint).

![Screenshot 2022-11-16 at 10 24 55 AM](https://user-images.githubusercontent.com/18638226/202221642-14fcda7e-2070-426c-858c-9914014fe42b.png)
![Screenshot 2022-11-16 at 10 25 05 AM](https://user-images.githubusercontent.com/18638226/202221685-7aec4858-7f83-43ae-8328-f8cdb648a719.png)

Reply:
Yea the issue is we don't currently have a backend endpoint for keeping track of likes and updating them, so  I have it as the default value from the backend for now. If someone wants to set up that route before tomorrow I can work on updating the like counter.